tests/autoload.php

```diff
@@ -0,0 +1,14 @@
+<?php
+function test_auto_loader($className)
```

Comment:
why not just use composer autoloader? use "autoload-dev" section

Reply:
example: https://github.com/eventum/rpc/blob/v4.3.0/composer.json#L21-L25

Reply:
Less code is always better. I will look into it and change it.